### PR TITLE
misc: npx -y @grafana/create-plugin@latest update

### PR DESCRIPTION
**Pull request by jkroepke**
I run `npx -y @grafana/create-plugin@latest update` on repository root.

In newer version of `@grafana/create-plugin` there is an supervisord which automatically rebuild the plugin on file change and attach a go debugger which make the development much more easier.

### CODE REVIEW

File: .config/Dockerfile [hunk 1]
```diff
@@ -1,8 +1,16 @@
 ARG grafana_version=latest
-ARG grafana_image=grafana-oss
+ARG grafana_image=grafana-enterprise
```
[the remark this answers is not in the document]
Comment (jkroepke) [reply .config/Dockerfile:2]:
```suggestion
ARG grafana_image=grafana-oss
```

File: .config/Dockerfile [hunk 2]
```diff
@@ -11,10 +19,60 @@ ENV GF_AUTH_BASIC_ENABLED "false"
...
+ENV GF_PATHS_HOME="/usr/share/grafana"
+WORKDIR $GF_PATHS_HOME
+
 USER root
 
-# Install chromium
 RUN apk add chromium
 
+# Installing supervisor and inotify-tools
+RUN if [ "${development}" = "true" ]; then \
```
Comment (mahendrapaipuri) [inline .config/Dockerfile:33]: When I test it locally, `development` is being set to `True` or `False`, notice the **capitalized**. Is it the same for you?

Unless I add `"${development}" = "True"` condition as well, it does not work

Comment (jkroepke) [reply .config/Dockerfile:33]: I set "true" (not capitalized) at the docker-compose.yaml and it worked fine.

File: docker-compose.yaml [hunk 3]
```diff
@@ -9,11 +7,19 @@ services:
       args:
         grafana_image: ${GRAFANA_IMAGE:-grafana-oss}
         grafana_version: ${GRAFANA_VERSION:-11.1.0}
+        development: false
```
Comment (jkroepke) [inline docker-compose.yaml:10]:
```suggestion
        development: ${DEVELOPMENT:-false}
```

File: docker-compose.yaml [hunk 4]
```diff
@@ -9,11 +7,19 @@ services:
...
         grafana_version: ${GRAFANA_VERSION:-11.1.0}
+        development: ${DEVELOPMENT:-false}
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - seccomp:unconfined
+    privileged: true
     ports:
       - 3000:${GF_SERVER_HTTP_PORT:-3000}/tcp
+      - "2345:2345/tcp"
```
Comment (jkroepke) [inline docker-compose.yaml:18]:
```suggestion
      - 2345:2345/tcp # delve
    security_opt:
      - "apparmor:unconfined"
      - "seccomp:unconfined"
```

File: docker-compose.yaml [hunk 5]
```diff
@@ -9,11 +7,19 @@ services:
       args:
         grafana_image: ${GRAFANA_IMAGE:-grafana-oss}
         grafana_version: ${GRAFANA_VERSION:-11.1.0}
+        development: ${DEVELOPMENT:-false}
+    cap_add:
+      - SYS_PTRACE
+    security_opt:
+      - seccomp:unconfined
```
Comment (jkroepke) [inline docker-compose.yaml:14]:
```suggestion
```

File: .config/Dockerfile (hunk 2 above)
Comment (mahendrapaipuri) [reply .config/Dockerfile:33]: I have them set with smalls as well, like `true`, `false`. But inside the container, they are being capitalized. I use podman and podman compose, probably that is the reason.

Comment (jkroepke) [reply .config/Dockerfile:33]: I'm using docker-compose and having no issues. But it sounds like your error is valid and should reported upstream

Comment (mahendrapaipuri) [reply .config/Dockerfile:33]: It seems to be a bug from `podman-compose`.

**Pull request merged**